spec/models/service_spec.rb

```diff
@@ -685,6 +685,18 @@
       service = FactoryGirl.build(:service, :retired => nil)
       expect(service).not_to be_valid
     end
+
+    context "retires service children" do
+      let(:service) { FactoryGirl.create(:service) }
```

Comment:
Any reason that these `let`s aren't inside the example?  Generally I only move things to a `let` if it is shared across examples.